src/components/forms/RangeInput/RangeInput.tsx

```diff
@@ -17,9 +19,30 @@ export const RangeInput = ({
   inputRef,
   ...inputProps
 }: RangeInputProps & JSX.IntrinsicElements['input']): React.ReactElement => {
-  // Range defaults to min = 0, max = 100, step = 1, and value = (max/2) if not specified.
-
   const classes = classnames('usa-range', className)
+  // input range defaults to min = 0, max = 100, step = 1, and value = (max/2) if not specified.
+  const defaultMin = 0
+  const defaultMax = 100
+  const { min, max, defaultValue } = inputProps
+  const ariaMin = min || defaultMin
```

Comment:
I think we should also handle a case where the consumer passes in their own `aria-` attributes as props.  We only want to use our handling as a fallback.

So  use here
 ```
const ariaMax = ariaMax || max || defaultMax
const ariaMin = ariaMin || max || defaultMax
```

Reply:
done